### DISCUSSION TITLE
How to create a task that is scheduled until the next day?

### DISCUSSION THREAD

**Discussion by shaidimaapi** [ivan-lednev/obsidian-day-planner#493] [Q&A]
I am trying to create a task that is scheduled until the next day. 

For example: - [ ] 23:20 - 01:20 Task A

The schedule for this task would be July 6 11:20PM until July 7 1:20AM.

**Comment by ivan-lednev** [comment]
Hello, this is currently not implemented.

**Comment by Zo-ey** [comment]
Hello, 
It looks like it is now possible to plan a task the way you wanted it @shaidimaapi . However, is it possible to plan it for more than 24 hours ? It does not seem to work the ways I try it.
For example: - [ ] 2025-03-04::23:20 -  2025-03-08::01:20 Task A

The schedule for this task would be April 4 11:20PM until April 8 1:20AM.

I know this is a _day_ planner plugin, but I couldn't find any other plugin with such feature :)

Thank you for your incredible work!
